### PR TITLE
A better design for update channel selector

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Respawn of https://github.com/nextcloud/server/pull/38922

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Comment by Jerome-Herbinet** [comment]
Testing is welcome

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -510,6 +507,9 @@ export default {
 			cursor: pointer;
 			margin-left: 3px;
 			display: inline-block;
+			padding: 10px;
+			border-radius: 10px;
+			border: 2px solid var(--color-border-dark);
```
Comment (susnux) [inline apps/updatenotification/src/components/UpdateNotification.vue:512]:
```suggestion
			border: 2px solid var(--color-border-dark);
```

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 2]
```diff
@@ -510,6 +507,9 @@ export default {
...
```
Comment (susnux) [inline apps/updatenotification/src/components/UpdateNotification.vue:512]: @szaimen are we using dark or max contrast? You did some changes on the vue library

Comment (szaimen) [reply apps/updatenotification/src/components/UpdateNotification.vue:512]: I think i used maxcontrast for the ncradiobutton design...

**Review by susnux — APPROVED** [review]
👍

**Review by jancborchardt — APPROVED** [review]
Looks good design-wise! :)

**Pull request merged**